maps/torch/structures/signs.dm

```diff
@@ -62,7 +62,7 @@
 	icon = 'maps/torch/icons/obj/solgov-decals.dmi'
 	icon_state = "solgovseal"
 
-/obj/structure/sign/ntefcrest
+/obj/structure/sign/solgovseal
 	name = "\improper NTEF Crest"
```

Comment:
missed this: NTEF Crest -> Sol Gov Crest?

Reply:
Yes!